spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe "Projects show page:" do
+  before(:each) do
+    # Challenges =================================================================
+    @recycled_material_challenge = Challenge.create!(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create!(theme: "Apartment Furnishings", project_budget: 1000)
+    # Projects =================================================================
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+    # Contestants =================================================================
+    @jay = Contestant.create!(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create!(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create!(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create!(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    ContestantProject.create!(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create!(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create!(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create!(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create!(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    ContestantProject.create!(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+  describe "When I visit a project's show page ('/projects/:id')" do
+    it "I see that project's name and material And I also see the theme of the challenge that this project belongs to." do
+      visit "/projects/#{@lit_fit.id}"
+      
+      expect(page).to have_content("#{@lit_fit.name}")
+      expect(page).to have_content("#{@lit_fit.material}")
+      expect(page).to have_content("#{@lit_fit.challenge.theme}")
+    end
+    
+    it "I see a count of the number of contestants on this project" do
+      visit "/projects/#{@news_chic.id}"
+      
+      expect(page).to have_content("Number of Contestants: #{@news_chic_count}")
```

Comment:
This is a nice robust assertion because it ensures that the contestant number is actually tied to the text, rather than just a floating number